bundle/tests/python_wheel/python_wheel_no_artifact/bundle.yml

```diff
@@ -8,8 +8,7 @@ resources:
       tasks:
         - task_key: TestTask
           existing_cluster_id: "0717-aaaaa-bbbbbb"
-          python_wheel_task:
```

Comment:
Should we keep both the tests instead to avoid losing coverage?

Reply:
Added